gammapy/background/tests/test_fov.py

```diff
@@ -1,2 +1,110 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
 from __future__ import print_function, division
+
+from gammapy.background import fill_acceptance_image
+from gammapy.image import make_empty_image
+
+from astropy.io import fits
+from astropy.coordinates import SkyCoord, Angle
+from astropy.wcs import WCS
+from astropy.wcs.utils import pixel_to_skycoord
+import astropy.units as u
+from astropy.units.quantity import Quantity
+
+import numpy as np
+from scipy.integrate import quad
```

Comment:
scipy is an optional dependency:
http://astropy.readthedocs.org/en/latest/development/testguide.html#tests-requiring-optional-dependencies
(search for `scipy` in test files if you want an example)

If you don't do this, test collection will fail for users that don't have `scipy`, which is not what we want.
We can discuss if `scipy` should become a required dependency for Gammapy in a separate issue, but for now, please fix this (or travis-ci will fail and the PR can't be merged)